src/app/dashboard/dashboard.component.css

```diff
@@ -1,6 +1,25 @@
-accordion-group{
+accordion-group {
     margin-bottom: 20px;
 }
-.heading{
+
+.heading {
     padding: 10px 0px;
-}
\ No newline at end of file
+}
+
+.container {
+  margin: 40px !important;
+}
+
+mat-expansion-panel-header {
+    background-color: #3f51b5 !important;
+}
+
+.mat-expansion-panel-header.mat-expanded:focus,
+.mat-expansion-panel-header.mat-expanded:hover {
+    background-color: #3f51b5;
```

Comment:
These color overrides should be done via the Angular Material theming system rather than hard coded overrides.